bugout-dev/moonstream
# Add "mooncrawl ethcrawler trending" command

## Changes

`mooncrawl ethcrawler trending` calculates trending addresses in a given time range from transactions mined into the Ethereum blockchain during that time.




## How to test these changes?

Check out this branch and run the following from `/crawlers`:

```bash
pip install -e .
mooncrawl ethcrawler trending -s $(date -d '-2 hours' +%Y%m%d-%H%M) -e $(date +%Y%m%d-%H%M) | jq .
```




## Related issues

https://github.com/bugout-dev/moonstream/issues/83
